src/animation.js

```diff
@@ -28,23 +31,26 @@ var _requestFrame = function(inCallback) {
 *
 * @private
 */
-var _cancelFrame = function(inId) {
-	return global.clearTimeout(inId);
+_cancelFrame = function(id) {
+	return global.clearTimeout(id);
 };
 
-for (var i = 0, pl = prefix.length, p, wc, wr; (p = prefix[i]) || i < pl; i++) {
+for (i = 0, pl = prefix.length, p, wcRAF, wrAF, wcAF; (p = prefix[i]) || i < pl; i++) {
```

Comment:
There's no need for p, wcRAF, wrAF, or wcAF here since they were declared above.